src/pages/workspace/accounting/qbo/export/QuickbooksCompanyCardExpenseAccountPage.tsx

```diff
@@ -19,7 +19,7 @@ import ROUTES from '@src/ROUTES';
 function QuickbooksCompanyCardExpenseAccountPage({policy}: WithPolicyConnectionsProps) {
     const {translate} = useLocalize();
     const styles = useThemeStyles();
-    const policyID = policy?.id ?? '-1';
+    const policyID = policy?.id ?? `${CONST.DEFAULT_NUMBER_ID}`;
```

Comment:
This still violates the link rule. Please check https://expensify.slack.com/archives/C01GTK53T8Q/p1737913879440099

Reply:
Ah thank you. I only recently learnt that the rule doesn't apply for strings.

Reply:
To be fair this was implemented not long after the rule was implemented 😄

Reply:
PR: https://github.com/Expensify/App/pull/55804